bgpd/bgpd.c

```diff
@@ -5999,16 +5999,20 @@ int peer_route_map_set(struct peer *peer, afi_t afi, safi_t safi, int direct,
 	struct peer *member;
 	struct bgp_filter *filter;
 	struct listnode *node, *nnode;
+	struct route_map *route_map = route_map_lookup_by_name(name);
 
 	if (direct != RMAP_IN && direct != RMAP_OUT)
 		return BGP_ERR_INVALID_VALUE;
 
+	if (!route_map)
+		return BGP_ERR_INVALID_VALUE;
```

Comment:
guys, should we only warn instead of returning an error here or how? because our tests rely on non-existing route-map https://github.com/FRRouting/frr/blob/master/tests/bgpd/test_peer_attr.c#L262 and so on.